pkg/roachprod/logger/log.go

```diff
@@ -126,8 +126,8 @@ func (cfg *Config) NewLogger(path string) (*Logger, error) {
 		return &Logger{
 			Stdout:  newSafeWriter(stdout),
 			Stderr:  newSafeWriter(stderr),
-			stdoutL: log.New(os.Stdout, cfg.Prefix, logFlags),
-			stderrL: log.New(os.Stderr, cfg.Prefix, logFlags),
+			stdoutL: log.New(stdout, cfg.Prefix, logFlags),
```

Comment:
Ran into the same issue in another (open) PR: https://github.com/cockroachdb/cockroach/pull/137874/files#diff-ee489974a5aec9076e3e38465697765dc6642512a454c42574e6354a8fc6efd4